addons/udes_stock/models/stock_picking_type.py

```diff
@@ -70,6 +70,12 @@ class StockPickingType(models.Model):
         help="When True, operations are automatically validated in real time.",
     )
 
+    u_handle_partials = fields.Boolean(
+        string="Process Partial Transfers",
+        default=True,
```

Comment:
We'll need to review what picking types we have setup should have this set to False, not for this story though.

Reply:
Yep - agreed with @ws-mps that we would just default to true for ow.